main.py

```diff
@@ -178,26 +178,25 @@ async def compile_project(project_root: Path, manifest: PackageManifest):
     # Create a contracts directory
     contracts_dir = project_root / "contracts"
     contracts_dir.mkdir()
-
     # add request contracts in temp directory
     if manifest.sources:
         for filename, source in manifest.sources.items():
             path = contracts_dir / filename
             # NOTE: In case there is a multi-level path
             path.parent.mkdir(parents=True, exist_ok=True)
             path.write_text(source.fetch_content())
-    with config.using_project(project_root) as project:
-        try:
-            compiled_manifest = project.extract_manifest()
-            results[project_root.name] = compiled_manifest
-            tasks[project_root.name] = TaskStatus.SUCCESS
-        except VyperCompileError as e:
-            results[project_root.name] = [
-                f"{e['sourceLocation'].get('file', 'Unknown file')}\n{e['type']}: {e.get('formattedMessage', e['message'])}"
-                for e in e.base_err.error_dict
-            ]
-            
-            tasks[project_root.name] = TaskStatus.FAILED
-        except Exception as e:
-            results[project_root.name] = {e.__class__.__name__:str(e)}
-            tasks[project_root.name] = TaskStatus.FAILED
+    project = Project(project_root)
```

Comment:
What happened to all that work we did using `Project.from_manifest(manifest_path)`

Reply:
it is now 
```